languages/python.md

```diff
@@ -23,6 +23,37 @@ def widgetize(widget):
   standardize(widget)
 ```
 
+## Linting
+It is recommended to use [black](https://black.readthedocs.io/en/stable/) (formatting), [flake8](https://flake8.pycqa.org/en/latest/index.html#quickstart) (formatting), [bandit](https://bandit.readthedocs.io/en/latest/) (security), and [isort](https://pycqa.github.io/isort/) (import order) during development and CI. Black is preferred as the primary formatting style due to its easy methods for auto-formatting the code. 
+
+To install these linters when starting a new project, run `pipenv install black flake8 bandit isort`. The linters can be run together with `make lint` after adding the following commands to a project's Makefile:
+
+```
+lint: bandit black flake8 isort
+
+bandit:
+	pipenv run bandit -r llama
+
+black:
+	pipenv run black --check --diff llama tests
+
+coveralls: test
+	pipenv run coveralls
+
+flake8:
+	pipenv run flake8 llama tests
+
+isort:
+	pipenv run isort llama tests --diff
+```
+
+
+[mypy](https://mypy.readthedocs.io/en/stable/) should be used whenever type hinting is desired.
```

Comment:
I think we should recommend both type hinting and the use of mypy.

Reply:
Sounds good to me!